src/com/owncloud/android/ui/activity/Uploader.java

```diff
@@ -444,6 +485,27 @@ private void populateDirectoryList() {
             mListView.setOnItemClickListener(this);
         }
     }
+    
+    public void startSyncFolderOperation(OCFile folder) {
+        long currentSyncTime = System.currentTimeMillis(); 
+        
+        mSyncInProgress = true;
+        
+        // perform folder synchronization
+        RemoteOperation synchFolderOp = new RefreshFolderOperation( folder,
+                                                                        currentSyncTime, 
+                                                                        false,
+                                                                        false,
+                                                                        false,
+                                                                        getStorageManager(),
+                                                                        getAccount(),
+                                                                        getApplicationContext()
+                                                                      );
+        synchFolderOp.execute(getAccount(), this, null, null);
+        
+        setSupportProgressBarIndeterminateVisibility(true);
```

Comment:
Ah, the progressbar has been implemented on the FileDisplayActivity and Uploader is extending FileActivity, so no Progressbar there :(

You might have to add the progress bar to your uploader layout and look up the element in populateDirectoryList()